dali/operators/color/brightness_contrast.cc

```diff
@@ -45,6 +45,11 @@ Additionally, this operator can change the type of data.)code")
     .AddOptionalArg("contrast",
                     "Set the contrast multiplier; 1.0 is neutral, 0.0 produces uniform grey.",
                     1.0f, true)
+    .AddOptionalArg("contrast_center",
+                    "Sets the instensity level that is unaffected by contrast - this is the value "
+                    "which all pixels assume when contrast is zero. When not set, the half of the "
+                    "input types's positive range (or 0.5 for float) is used.",
+                    0.5f, false)
```

Comment:
It would be nice to put here an example, when the user would like to specify this param. Cause at the first glance it's not obvious for me, when I should use it